pkg/zsio/zeek/parser.go

```diff
@@ -246,9 +247,18 @@ func (p *Parser) ParseValue(line []byte) (*zson.Record, error) {
 		// each time here
 		path = []byte(p.path)
 	}
-	zv, ts, err := zson.NewRawAndTsFromZeekTSV(p.builder, p.descriptor, path, line)
+	zv, tsVal, err := zson.NewRawAndTsFromZeekTSV(p.builder, p.descriptor, path, line)
 	if err != nil {
 		return nil, err
 	}
+	var ts nano.Ts
```

Comment:
This conversion of unset to `0` is done to maintain current behavior. But I would suggest that we change the record type,  making  `record.Ts` a `zeek.Value` (rather than a `nano.Ts as currently). This would remove the current ambiguity and allow the runtime to distinguish between an unset time and a zero-valued timestamp.